models/public/efficientdet-d0-tf/efficientdet-d0-tf.md

```diff
@@ -0,0 +1,82 @@
+# efficientdet-d0-tf
+
+## Use Case and High-Level Description
+
+The "efficientdet-d0" model is one of the [EfficientDet](https://arxiv.org/abs/1911.09070)
+models  designed to perform object detection. This model was pretrained in TensorFlow*.
+All the EfficientDet models have been pretrained on the MSCOCO* image database.
+For details about this family of models, check out the Google AutoML [repository]
+(https://github.com/google/automl/tree/master/efficientdet).
+
+## Specification
+
+| Metric            | Value           |
+|-------------------|-----------------|
+| Type              | Object detection|
+| GFLOPs            |     -           |
+| MParams           |     -           |
+| Source framework  | TensorFlow\*    |
+
+## Accuracy
+
+| Metric | Converted model |
+| ------ | --------------- |
+| coco_precision | 31.95%|
```

Comment:
The `coco_precision` is a name from AC. I think, it is better to use more common name, such as AP50, may be with link to  metric description.